decidim/decidim
# Add confirmation email with answers in survey questionnaires

#### :tophat: What? Why?
This PR adds a new setting in the surveys component with which an email is sent to the user after answering a questionnaire.

The email contains a confirmation with the user answers.

#### Testing
1. Create a questionnaire and publish the component
2. Answer questionnaire
3. Check LetterOpener with confirmation email

### :camera: Screenshots
![imagen](https://github.com/decidim/decidim/assets/75725233/1df896ce-8f28-4b51-944c-bfce46aa7882)
![imagen](https://github.com/decidim/decidim/assets/75725233/c210f2a6-7aeb-4c07-96c0-bf94691622cf)
![imagen](https://github.com/decidim/decidim/assets/75725233/bb8ceb37-4713-42bf-935b-223214b0b6d5)


:hearts: Thank you!
